examples/einsum.py

```diff
@@ -19,9 +19,14 @@
 import re
 
 from benchmark import run_benchmark
-from legate.timing import time
 
-import cunumeric as np
+try:
+    from legate.timing import time
+except ImportError:
+    from time import perf_counter_ns
+
+    def time():
+        return perf_counter_ns() / 1000.0
```

Comment:
If we will return seconds, we could just as well use `perf_counter`.

Reply:
Use perf_counter * 1000000.0 instead.

Reply:
Ah, sorry. My bad, but this does not make much sense. For some reason, I was thinking we are measuring seconds, but we are measuring miliseconds, so `perf_counter_ns` makes more sense. Sorry. :)

Reply:
Fixed. Thanks.